lib/log_format.c

```diff
@@ -536,6 +536,33 @@ qb_vsnprintf_serialize(char *serialize, size_t max_len,
 				format++;
 			}
 			goto reprocess;
+		case 'z':
+			format++;
+			if (sizeof(size_t) == sizeof(long long)) {
+				type_longlong = QB_TRUE;
+			}
+			else {
+				type_long = QB_TRUE;
+			}
+			goto reprocess;
+		case 't':
+ 			format++;
```

Comment:
Please, do not mix spaces and tabs.
`git diff` (or perhaps `git diff --color`) should expose that visually
(using `color.diff.whitespace`) by default.